src/filters/simple-filter-fields.html

```diff
@@ -0,0 +1,32 @@
+<div class="simple-filter">
+  <form>
+    <div class="form-group">
+      <div class="input-group">
+        <div dropdown class="input-group-btn">
+          <button dropdown-toggle type="button" class="btn btn-default dropdown-toggle filter-fields" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            {{currentField.title}}
+            <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu">
+            <li ng-repeat="item in config.fields">
+              <a class="filter-field" role="menuitem" tabindex="-1" ng-click="selectField(item)">
+                {{item.title}}
+              </a>
+            </li>
+          </ul>
+        </div>
+        <input class="form-control" type="{{currentField.filterType}}" ng-model="config.currentValue"
+               placeholder="{{currentField.placeholder}}"
+               ng-keypress="onValueKeyPress($event)"
+               ng-if="currentField.filterType !== 'select'"/>
+        <select pf-select class="form-control filter-select" id="currentValue"
+                ng-model="config.currentValue"
+                ng-options="o as o for o in currentField.filterValues"
```

Comment:
It needs to have the select and the value but not the as. I agree this would be more readable if simply o was not used. I will update.